File: CareerCrafterAngular/src/app/models/ApplicationReaddto.ts
import { ResumeReadDto } from "./ResumeReaddto";

export interface ApplicationReadDto {
    applicationId: number;
    jobSeekerId: number;
    jobListingId: number;
    appliedDate: Date;
    status: string;
    jobTitle: string;          
    jobSeekerName: string;
    companyName : string;
    resume: ResumeReadDto | null;
}

File: CareerCrafter/CareerCrafter.DTOs/ApplicationDTOs/ApplicationReadDto.cs
using CareerCrafter.DTOs.ResumeDTOs;

namespace CareerCrafter.DTOs.ApplicationDTOs
{

    public class ApplicationReadDto
    {
        public int ApplicationId { get; set; }
        public int JobSeekerId { get; set; }
        public int JobListingId { get; set; }
        public DateTime AppliedDate { get; set; }
        public string Status { get; set; }

        public string JobTitle { get; set; }
        public string companyName { get; set; }

        public ResumeReadDto Resume { get; set; }
    }
}
